src/main/java/core/basesyntax/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        boolean isFirstLine = true;
+        for (String line : data) {
+            if (isFirstLine) {
+                isFirstLine = false;
+                continue;
+            }
+            String[] parts = line.split(",");
+            if (parts.length != 3) {
+                continue;
```

Comment:
Consider using a constant for the expected number of parts after splitting the line to improve the readability of the code and avoid magic numbers.